src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -305,7 +305,6 @@ function createOpenTagMarkup(
   if (isRootElement) {
     ret += ' ' + DOMMarkupOperations.createMarkupForRoot();
   }
-  ret += ' ' + DOMMarkupOperations.createMarkupForID('');
```

Comment:
I think this changes the behavior if you try to `ReactDOM.render` into a non-container. But maybe that's fine?